.github/workflows/main.yml

```diff
@@ -23,6 +23,11 @@ jobs:
                         operating-system: windows-latest
                         coverage: xdebug
 
+                    -   php-versions: ['8.2', '8.3']
```

Comment:
I'm not sure whether this will work as array

Reply:
Nope :-) Fixed in new commit